src/elements/message/message.scss

```diff
@@ -26,7 +27,8 @@
 }
 
 ::slotted([slot='image']) {
-  margin-block: var(--sbb-message-image-margin-block);
+  display: block;
+  margin-block: var(--sbb-message-image-margin-block) !important; // overrides '.sbb-figure' margin
```

Comment:
hmm. In theory the consumer could set a custom margin block which would be overridden from our definitions. otherwise, the design gives this margin as given and therefore shouldn't be overridden. A potential alternative would be to use :where() on the figure styles. But not convinced as well. So my comment is probably completely useless :D

Reply:
The only 'viable' alternative to remove the '!important' is to add in core
```css
  > figure[slot='image'] {
    margin-block: var(--sbb-message-image-margin-block)
  }
```